src/components/CertificateDropZone/Views/UnverifiedView.test.js

```diff
@@ -42,14 +44,16 @@ const STATUS = ["HASH", "ISSUED", "REVOKED", "IDENTITY"];
 describe("UnverifiedView", () => {
   it("displays hash error if the hash is invalid", () => {
     const wrapper = mount(
-      <UnverifiedView
-        handleRenderOverwrite={() => {}}
-        verificationStatus={{
-          ...VALID_VERIFICATION_STATUS,
-          hash: { checksumMatch: false }
-        }}
-        resetData={() => {}}
-      />
+      <MemoryRouter>
```

Comment:
Link component needs a router to be declared otherwise it throws an error, memory router is the simplest to use in test (can also test navigation that way)

Reply:
Ahh got it, thanks!